System/Win32/Console.hsc

```diff
@@ -63,21 +65,15 @@ module System.Win32.Console (
 #include "wincon_compat.h"
 
 import System.Win32.Types
+import System.Win32.Console.Internal

```

Comment:
Thanks, looks good, I thibk we'd want to re-export this new module though I wouldn't want to break existing code.

Reply:
We already re-export all the symbols